src/main/java/io/github/lottetreg/httpserver/Server.java

```diff
@@ -0,0 +1,10 @@
+package io.github.lottetreg.httpserver;
+
+public class Server {
```

Comment:
Most of the classes are public in the project thus far, I think some of them could probably be package-private. You could wait to refactor those but something to start thinking about (along with package principles).

Reply:
(More on package principles: https://medium.com/@andrewMacmurray/package-principles-850d89da5e5b. See also PPP)

Reply:
I forgot to thank you both for this! I'll read up on package principles :)